logistics_project/apps/malawi/warehouse/report_views/dashboard.py

```diff
@@ -68,6 +70,58 @@ def get_stockout_rates_table(self, request, child_sps, window_date):
 
         return table
 
+    def get_district_fridge_summary_table(self, request, reporting_supply_point, child_sps):
+        if request.base_level_is_facility:
```

Comment:
just a style thing, but if the whole function is in the `if` i find it can be easier to read if you put the opposite condition in the return at the beginning (saw you did this in the next commit for a different function)